articles/machine-learning/resource-known-issues.md

```diff
@@ -425,6 +425,16 @@ method, or from the Experiment tab view in Azure Machine Learning studio client
   2. Enter `pip freeze` and look for `tensorflow`, if found, the version listed should be < 1.13
   3. If the listed version is a not a supported version, `pip uninstall tensorflow` in the command shell and enter y for confirmation.
 
+## Model explanations
+
+* **Sparse data not supported**: The model explanation dashboard breaks/slows down substantially with a large amount of features, therefore we currently do not support sparse data format. Additionally, general memory issues will arise with large datasets and large number of features. 
+
+* **Forecasting models not supported with model explanations**: Interpretability, best model explanation, is not available for Auto ML forecasting experiments that recommend the following algorithms as the best model: ForecastTCN, Average, Naive, Seasonal Average, and Seasonal Naive.
```

Comment:
Nit: "Auto ML" should either be "Automated ML" or "AutoML" depending on what the docs team recommends

Reply:
LOL this was also pulled from the AutoML docs... I'll just go with AutoML